### PR TITLE
New feature

### PR DESCRIPTION
**Pull request by saurabh077**
This is new feature

### CODE REVIEW

File: new feature.c [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#include<stdio.h>
+int main(){
+	printf("This is new feature");
```
Comment (riteshja88) [inline new feature.c]: can we print `Hello World!`?

Comment (saurabh077) [reply new feature.c]: Done

Comment (riteshja88) [reply new feature.c]: can we remove print `This is new feature`?

Comment (saurabh077) [reply new feature.c]: yes. done.

**Review by riteshja88 — APPROVED** [review]

**Pull request merged**